ethereum/sidecar/server.go

```diff
@@ -1026,6 +1037,20 @@ func (s *Server) attestAssetsUnlockedEvents(ctx context.Context) {
 					continue
 				}
 
+				// first try with the batch attestation stuff
+				// ok, err = s.batchAttestation.TryAttest(ctx, bridgeAssetsUnlocked)
+				// if err != nil {
+				// 	if err == ctx.Err() {
+				// 		attestationLogger.Info("stopping attestation slot wait due to context cancellation")
+				// 	}
+				// 	attestationLogger.Error("batch attestation terminated with error", "attestation", attestation, "error", err)
+				// }
+				// if ok {
```

Comment:
Wondering if we should wait 32 until the batch submission lands on the safe epoch. First thought was that we shouldn't because it's the Bridge Worker who is in charge of submitting the batch attestation and re-submit it upon need. However, what if the Bridge Worker just goes down after submitting and the transaction gets reorged? Validators should be able to fallback into the individual mode in that case. On the other hand, I don't like adding the 32-block delay here and artificially make the processing time longer. That makes me think we should have a separate process catching up such deviations and putting them to the attestation queue again. This way, we can also get rid of the 32-block wait during individual attestations. Of course, not for this PR but curious about your thoughts.